File: src/main/java/com/bpzj/auto/deploy/services/MyProjectService.java
package com.bpzj.auto.deploy.services;

import com.intellij.openapi.project.Project;
import com.bpzj.auto.deploy.MyBundle;
import org.jetbrains.annotations.NotNull;


public class MyProjectService {
    public MyProjectService(@NotNull Project project) {
        System.out.println(MyBundle.message("projectService", project.getName()));
    }
}

File: src/main/kotlin/com/github/bpzj/autodeploy/services/MyProjectService.kt
package com.github.bpzj.autodeploy.services

import com.intellij.openapi.project.Project
import com.github.bpzj.autodeploy.MyBundle

class MyProjectService(project: Project) {

    init {
        println(MyBundle.message("projectService", project.name))
    }
}
